### PR TITLE
PLT-9208 PLT-9206 PLT-9207 PLT-9212 PLT-9213 Revise nightly tests to use `marlowe-benchmark`

### PR DESCRIPTION
**Pull request by bwbush**
1. Removes `marlowe-scaling`.
2. Removes `marlowe-finder`.
3. Updates operables and OCI images add `marlowe-benchmark`.
4. Updates nightly tests to use `marlowe-benchmark` instead of `marlowe-scaling` and `marlowe-finder`.
5. Revises documentation on stress and scale testing.
6. Provides examples of running benchmarks in a non-SRE environment.

See example results at https://ipfs.functionally.io/ipfs/QmcHidP9pbK7aw7uqFsfoM4jMYom8JPhqJHLrncfRB7MqE.

Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, PNG optimization, etc. are updated
    - [x] Operables are updated with changes to executable command line options.
    - [x] Deploy charts updated with changes to operables.
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
        - Review required
        - [x] Substantial changes to code, test, or documentation
        - [ ] Change made to Marlowe validator (@bwbush and @palas must be included as reviewers)
        - Review not required
        - [ ] Minor changes to non-critical code, documentation, nix derivations, configuration files, or scripts
        - [ ] Formatting, spelling, grammar, or reorganization
    - [x] Reviewer requested

### CODE REVIEW

**Comment by jhbertra** [comment]
Why do we need OCI images for marlowe benchmark?

**Comment by bwbush** [comment]
> Why do we need OCI images for marlowe benchmark?

We'll be running the tests in a container. In fact, this PR will revise the nightly tests to use some of the benchmarks in preference to `marlowe-finder` and `marlowe-scaling`, which can be retired.

**Comment by bwbush** [comment]
@renebarbosafl and/or @shlevy, here are the nightly build lines that need your scrutiny: https://github.com/input-output-hk/marlowe-cardano/blob/1a11e2d49b582ceb825d07bfc08ade188871fc92/.github/workflows/nightly.yml#L77-L86.

**Review by renebarbosafl — APPROVED** [review]

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -105,28 +72,15 @@ jobs:
...
-            "$(cat test-wallets/test.11.addr)=test-wallets/test.11.skey" \
-            "$(cat test-wallets/test.12.addr)=test-wallets/test.12.skey" \
-            "$(cat test-wallets/test.13.addr)=test-wallets/test.13.skey" \
-            "$(cat test-wallets/test.14.addr)=test-wallets/test.14.skey" \
-            "$(cat test-wallets/test.15.addr)=test-wallets/test.15.skey" \
-            "$(cat test-wallets/test.16.addr)=test-wallets/test.16.skey" \
-            "$(cat test-wallets/test.17.addr)=test-wallets/test.17.skey" \
-            "$(cat test-wallets/test.18.addr)=test-wallets/test.18.skey" \
-            "$(cat test-wallets/test.19.addr)=test-wallets/test.19.skey"
+          mkfifo node.socket
```
Comment (shlevy) [inline .github/workflows/nightly.yml:77]: This is superfluous, `socat` will create its own unix socket (which is not the same as a fifo)

**Review by shlevy — APPROVED** [review]

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (bwbush) [reply .github/workflows/nightly.yml:77]: Makes sense. Thanks!

**Pull request merged**